gui.js

```diff
@@ -308,6 +308,14 @@ IDE_Morph.prototype.openIn = function (world) {
         }
     }
 
+    // This function returns the value of a parameter given its key
+    function getParameterByName(name) {
```

Comment:
Oh, you mean https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/blob/master/cloud.js#L607? I didn't know about that, I'll change it!